tests/js/server/aql/aql-optimizer-edge-index.js

```diff
@@ -202,9 +202,9 @@ function optimizerEdgeIndexTestSuite () {
 
       queries.forEach(function (query) {
         var results = AQL_EXECUTE(query[0]);
-        assertEqual(query[1], results.json.length, query[0]);
-        assertEqual(0, results.stats.scannedFull);
-        assertEqual(query[1], results.stats.scannedIndex);
+        assertEqual(query[1], results.json.length, query[0], "Query: " + query + " - " + JSON.stringify(results));
```

Comment:
@dothebart You forgot to remove `, query[0]`, currently this change has no effect.